apps/studio/utils/portable-text-validations.ts

```diff
@@ -0,0 +1,177 @@
+import {
```

Comment:
So I actually got this from a Sanity YouTube video https://www.youtube.com/watch?v=KGnuq59XbRg